crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/meta_net.rs

```diff
@@ -357,10 +435,23 @@ impl MetaNetCon {
                             let wrap = WireWrap::request(msg_id, WireData(wire));
                             let data = wrap.encode_vec().map_err(KitsuneError::other)?;
 
+                            let mut metric_guard = MetricSendGuard::new(
+                                metric_msg_out_byte.clone(),
+                                metric_msg_out_time.clone(),
+                                rem_url.id().unwrap(),
+                                data.len() as u64,
+                            );
+
+                            tracing::warn!(%rem_url, "send data");
```

Comment:
why is this a warning?

Reply:
Whoops, good catch! That was for debugging, I'll fix that.